cryptostore/aggregator/aggregator.py

```diff
@@ -76,7 +77,7 @@ def run(self):
                     interval_start = aggregation_start
                     if end:
                         interval_start = end + timedelta(seconds=interval + 1)
-                    start, end = get_time_interval(interval_start, interval, multiplier=multiplier)
+                    start, end = get_time_interval(interval_start, base_interval, multiplier=multiplier)
```

Comment:
see above comment - this should have no change

Reply:
Same as above. There is a difference in behavior.